.prettierignore

```diff
@@ -0,0 +1 @@
+package.json
```

Comment:
Have to ignore the `package.json`, otherwise `prettier` will format it with tabs